Travel Project v1/Travel Project v1/Program.cs

```diff
@@ -0,0 +1,72 @@
+﻿using Event;
+using InterfaceImplementation;
+using Interfaces;
+using SomeStuff;
+using System;
+
+namespace Travel_Project_v1
+{
+    class Program
+    {
+
+        static void Main(string[] args)
+        {
+
+            //Event
+
+            double d = double.Parse(Console.ReadLine());
+            var priceFlight = new EventLogic();
+            EventHandling showPrice = new EventHandling();
+            showPrice.ShowTotalFlightPrice(d);
+            priceFlight.PriceOfFlight(d);
+
+            PriceDependingDistanceCalculation?.Invoke(PriceOfFlight(d));
```

Comment:
Noticed that this would not compile at all, I am a bit dissapointed that you provided me for review solution that does not compile. I would assume that this was done by mistake and hope to see a fixed version.